cmd/podman/common/build.go

```diff
@@ -239,6 +236,36 @@ func ParseBuildOpts(cmd *cobra.Command, args []string, buildOpts *BuildFlagsWrap
 		default:
 			return nil, fmt.Errorf("no Containerfile or Dockerfile specified or found in context directory, %s: %w", contextDir, syscall.ENOENT)
 		}
+	} else {
+		for _, f := range containerFiles {
+			if isURL(f) || f == "/dev/stdin" {
+				continue
+			}
+
+			// If it is a directory, skip it and let buildah handle the situation
+			// Because currently the podman CI runs the test script buildah/test/bud.bats,
+			// which checks if the error is a certain string. If we handle it here,
+			// it will cause the buildah tests to never pass (unless we use the same string).
+			// See: https://github.com/containers/buildah/blob/4c781b59b49d66e07324566555339888113eb7e2/imagebuildah/build.go#L139-L141
+			// 	    https://github.com/containers/buildah/blob/4c781b59b49d66e07324566555339888113eb7e2/tests/bud.bats#L3474-L3479
+			//      https://api.cirrus-ci.com/v1/artifact/task/4705206577922048/html/bud-remote-fedora-40-root-host-sqlite.log.html
```

Comment:
This cirrus link will disappear eventually, these artifact links will not be kept forever I think.
If this should have a specific error than please write that error down in the comment.